BitmovinAnalyticsCollector/Classes/BitmovinPlayer/Classes/BitmovinPlayerAdapter.swift

```diff
@@ -52,9 +52,20 @@ class BitmovinPlayerAdapter: NSObject, PlayerAdapter {
         if let sdkVersion = Bundle(for: BitmovinPlayer.self).object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String {
             eventData.version = PlayerType.bitmovin.rawValue + "-" + sdkVersion
         }
-
-        // streamForamt, hlsUrl
-        eventData.streamForamt = "hls"
+        
+        let sourceUrl = player.config.sourceItem?.url(forType: player.streamType)
+        switch player.streamType {
+        case .DASH:
+            eventData.streamFormat = StreamType.dash.rawValue
+            eventData.mpdUrl = sourceUrl?.absoluteString
+        case .HLS:
+            eventData.streamFormat = StreamType.hls.rawValue
+            eventData.m3u8Url = sourceUrl?.absoluteString
+        case .progressive:
```

Comment:
why is `HLS` and `DASH` uppercase and  `.progressive` lowercase?

Reply:
Looks like different `enum`s... Can we unify them? Or maybe use one or another?

Reply:
that comes from the bitdash-ios-sdk, not sure why they named it this way. That's also the reason why we can't unify - the second `enum` is how we handle (or at least name the type) in analytics - should be the same on all platforms.